Modelica/Blocks/Sources.mo

```diff
@@ -1865,6 +1865,13 @@ When the constant \"NO_FILE_SYSTEM\" is defined, all file I/O related parts of t
 source code are removed by the C-preprocessor, such that no access to files takes place.
 </p>
 <p>
+If the table is read from a CSV file, the following limitations apply
+</p>
+<ol>
+<li>Non-numeric data is not supported (in the lines following the header lines), even if such columns are excluded.</li>
+<li>Double-quoted data entries in the first header line shall not contain the column delimiter.</li>
```

Comment:
Ok, so an entry like "first, entry" is not allowed for for `delimiter=","`, right? (I had to read this twice to understand.)

Reply:
Yes, this should be best avoided.